easybuild/tools/docs.py

```diff
@@ -707,6 +708,44 @@ def list_software_txt(software, detailed=False):
     return '\n'.join(lines)
 
 
+def gather_reverse_dependencies():
+    """
+    Gather reverse dependencies from installed modules.
+
+    :return: dictionary with reverse dependencies
+    """
+    reverse_dependencies = {}
+
+    module_avail_cmd = "module -t avail 2>&1 | grep -v ':$'"
```

Comment:
How would this work with a hierarchical scheme (or any scheme that involved extending the module path)?

Reply:
That shouldn't be a problem when using `modules_tool().available()`, which spits out a list of *full* module names in a hierarchy (i.e. things like `Core/GCC/6.4.0` rather than just `GCC/6.4.0`), so you'll see *all* modules regardless of where they are in the hierarchy...